test/scala_fixtures/no_active_sbt_project/Main.scala

```diff
@@ -0,0 +1,5 @@
+object HelloAle {
+  def main(args: Array[String]): Int = {
+    println("Hello, ale!")
+  }
+}
```

Comment:
You can delete this file and the other files. The tests won't read them, and we shouldn't make the `readfile()` call.